packages/p2p/src/components/buy-sell/buy-sell-form.jsx

```diff
@@ -261,71 +278,80 @@ const BuySellForm = props => {
-                                                }
-                                                hint={
-                                                    <Localize
-                                                        i18n_default_text='Limits: {{min}}–{{max}} {{currency}}'
-                                                        values={{
-                                                            min: min_order_amount_limit_display,
-                                                            max: max_order_amount_limit_display,
-                                                            currency: buy_sell_store.account_currency,
-                                                        }}
-                                                    />
-                                                }
-                                                is_relative_hint
-                                                className='buy-sell__modal-field'
-                                                trailing_icon={
-                                                    <Text color='less-prominet' line-height='m' size='xs'>
-                                                        {buy_sell_store.account_currency}
-                                                    </Text>
-                                                }
-                                                onKeyDown={event => {
-                                                    if (!floatingPointValidator(event.key)) {
-                                                        event.preventDefault();
+                                <div className='buy-sell__modal-line' />
+                                <div className='buy-sell__modal--input'>
+                                    <Text color='less-prominent' size='xxs'>
+                                        {localize('Enter {{transaction_type}} amount', {
+                                            transaction_type: buy_sell_store.is_buy_advert ? 'buy' : 'sell',
+                                        })}
+                                    </Text>
+                                    <section className='buy-sell__modal--input-field'>
+                                        <Field name='amount'>
+                                            {({ field }) => (
+                                                <Input
+                                                    {...field}
+                                                    data-lpignore='true'
+                                                    type='number'
+                                                    error={errors.amount}
+                                                    label={
+                                                        buy_sell_store.is_buy_advert
+                                                            ? localize('Buy amount')
+                                                            : localize('Sell amount')
```

Comment:
Perhaps to shorten the ternary, we could implement it like this?

```
localize(`${buy_sell_store.is_buy_advert ? 'Buy amount' : 'Sell amount}`)
```

Reply:
Will refactor the code